tests/publiccloud/storage_perf.pm

```diff
@@ -68,7 +68,7 @@ sub analyze_previous_series {
 
 
 sub run {
-    my ($self) = @_;
+    my ($self, $args) = @_;
```

Comment:
Shouldn't it be?

```suggestion
    my ($self, %args) = @_;
```

Reply:
No. It's always `$self` and `$args` in `tests/publiccloud/*.pm`